amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-examples.adoc

```diff
@@ -1,11 +1,6 @@
 = Amazon S3 Connector 5.7 Examples
 :page-aliases: connectors::amazon/amazon-s3-connector-examples.adoc
 
-The following are common use cases for the Amazon S3 connector:
-
-* Store an image from a URL on Amazon S3, and then retrieve and display the image.
-* Create an image link in Amazon S3 and update the status in Twitter along with the image link (not shown in this document).
-
 [[store-and-retrieve]]
 == Use Case: Store and Retrieve an Image from a URL to Amazon S3
```

Comment:
Please add the text "Paste this code..." before the XML. This is in the template for the Example topic.

Reply:
I see a few more issues in the Examples topic: